server/container_create_linux_test.go

```diff
@@ -159,6 +160,47 @@ func TestAddOCIBindsCGroupRW(t *testing.T) {
 	}
 }
 
+func TestAddOCIBindsErrorWithoutIDMap(t *testing.T) {
+	ctr, err := container.New()
+	if err != nil {
+		t.Error(err)
```

Comment:
Do we want t.Fatal() here, so the execution doesn't continue?

```suggestion
		t.Fatal(err)
```